website/server/controllers/api-v3/world.js

```diff
@@ -0,0 +1,47 @@
+import {
+  model as Group,
+  TAVERN_ID as tavernId,
+} from '../../models/group';
+
+let api = {};
+
+async function getWorldBoss () {
+  let tavern = await Group
+    .findById(tavernId)
+    .select('quest.progress quest.key quest.active quest.extra')
+    .exec();
+  let quest = {};
+  if (tavern) {
+    if (tavern.quest.active) {
+      quest = tavern.quest;
```

Comment:
Just return here rather than setting the variable at the top. So just return tavern.quest.